lib/src/data/chat/chat_rooms_create_request.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'chat_rooms_create_request.freezed.dart';
+part 'chat_rooms_create_request.g.dart';
+
+@freezed
+class ChatRoomsCreateRequest with _$ChatRoomsCreateRequest {
+  const factory ChatRoomsCreateRequest({
+    required String name,
+    required String description,
```

Comment:
```suggestion
    String? description,
```

ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/server/api/endpoints/chat/rooms/create.ts#L45